avro2tf/src/main/scala/com/linkedin/avro2tf/configs/Codecs.scala

```diff
@@ -0,0 +1,23 @@
+package com.linkedin.avro2tf.configs
+
+import io.circe.{Decoder, Encoder}
+import io.circe.generic.extras.Configuration
+
+/**
+  * Implicits for Circe.
```

Comment:
Do you think we can rename this file `Codecs.scala` to a more meaningful one?

Reply:
I'm renaming it to `JsonCodecs`, which is a bit better.